phpstan.neon.dist

```diff
@@ -57,3 +57,9 @@ parameters:
 			message: "#^Parameter \\#1 \\$callback of function call_user_func_array expects callable\\(\\): mixed, array\\{.+\\} given\\.$#"
 			count: 1
 			path: include/model.php
+
+		# Ignored because PHPStan seems confused about WP_REST_Request's template.
```

Comment:
I think it legit, indeed `$request['code']` *can* return `null`.
It's the same as doing `$request->get_param( 'code ' )`, which returns `null` if `code` isn't set in the request.

Reply:
It can't be `null` if it's in the route. Right?

Reply:
Well, we have to be sure to have a 404 error if we do a `PUT` request on `pll/v2/languages/` without code.
Iff so, I agree with you.

Reply:
I added some failure tests in [58e7fa9](https://github.com/polylang/polylang/pull/1505/commits/58e7fa9aada7a5433e1e6965baee94457812b418).